app-backend/datamodel/src/main/scala/SceneToLayer.scala

```diff
@@ -0,0 +1,45 @@
+package com.rasterfoundry.datamodel
+
+import geotrellis.vector.{Geometry, Projected}
+import io.circe.generic.JsonCodec
+import io.circe.Json
+
+import java.util.UUID
+
+final case class SceneToLayer(sceneId: UUID,
+                              projectLayerId: UUID,
+                              /* Has a Scene been accepted by a user?
+                               * TRUE if a user added this Scene manually as usual from the UI, or if
+                               * they accepted this Scene from a list of "pending" Scenes which passed an AOI
+                               * check. Defaults to FALSE when added by an AOI check via a batch process.
+                               *
+                               * Scenes marked FALSE here should not appear during normal tiling
+                               * activities in the UI.
+                               */
+                              accepted: Boolean,
+                              sceneOrder: Option[Int] = None,
+                              colorCorrectParams: ColorCorrect.Params)
+
+final case class SceneToLayerWithSceneType(
+    sceneId: UUID,
+    projectId: UUID,
+    projectLayerId: UUID,
+    accepted: Boolean,
+    sceneOrder: Option[Int] = None,
+    colorCorrectParams: ColorCorrect.Params,
+    sceneType: Option[SceneType] = None,
+    ingestLocation: Option[String],
+    dataFootprint: Option[Projected[Geometry]],
+    isSingleBand: Boolean,
+    singleBandOptions: Option[Json]
+)
+
+// @JsonCodec
```

Comment:
If these aren't necessary they should be removed

Reply:
They are removed in the `SceneToProject.scala` and they will reside in here, since we are deleting the former in the future.